src/arm64/executor/main.c

```diff
@@ -42,9 +42,14 @@ int (*set_memory_nx)(unsigned long, int) = 0;
 long uarch_reset_rounds = UARCH_RESET_ROUNDS_DEFAULT;
 char pre_run_flush = PRE_RUN_FLUSH_DEFAULT;
 char enable_faulty_page = ENABLE_FAULTY_DEFAULT;
-char *measurement_template = (char *)&template_l1d_flush_reload; // template_l1d_prime_probe is an alternative but is currently non-functional
 char *measurement_code = NULL;
 
+#if defined(CACHE_TIMING_ATTACK_FLUSH_RELOAD)
```

Comment:
The module already has a function for selecting the attack type (see `measurement_mode_store`), so the preprocessor macro duplicates this functionality. You can find a reference implementation in the `x86` directory

Reply:
Ah, thank you for pointing this out - I will update `measurement_mode_store` to look for specific characters in the buffer to select an attack type, like I see in the x86 executor.